### PR TITLE
Added tests for covering MyTested.AspNetCore.Mvc.Authentication

### PR DESCRIPTION
**Pull request by Bozhidar4**
Added unit tests to MyTested.AspNetCore.Mvc.Authentication.Test for covering MyTested.AspNetCore.Mvc.Authentication project.

### CODE REVIEW

File: test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs [hunk 1]
```diff
@@ -63,5 +64,104 @@ public void WithAuthenticatedUserShouldPopulateProperUserWhenUserWithUserBuilder
...
+                    Assert.True(user.Identity.IsAuthenticated);
+                });
+        }
+
+        [Fact]
+        public void WithAuthenticatedUserShouldPopulateIdAndUserNameAndRoleProperly()
+        {
+            MyViewComponent<UserComponent>
+                .Instance()
+                .WithUser("IdentityIdentifier", "NewUserName",
```
Comment (ivaylokenov) [inline test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs]: Fix these parameters. They should be like the one above.

File: test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs [hunk 2]
```diff
@@ -63,5 +64,104 @@ public void WithAuthenticatedUserShouldPopulateProperUserWhenUserWithUserBuilder
...
+                    Assert.True(user.Identity.IsAuthenticated);
+                });
+        }
+
+        [Fact]
+        public void WithAuthenticatedUserShouldPopulateUserNameAndRoleProperly()
+        {
+            MyViewComponent<UserComponent>
+                .Instance()
+                .WithUser("NewUserName",
```
Comment (ivaylokenov) [inline test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs]: Fix these parameters. They should be like:
```
                .WithUser("NewUserName", new List<string>
                {
                      "Administrator"
                })
```

File: test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs (hunk 1 above)
Comment (Bozhidar4) [reply test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs]: Parameters are fixed.

File: test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs (hunk 2 above)
Comment (Bozhidar4) [reply test/MyTested.AspNetCore.Mvc.Authentication.Test/BuildersTests/ViewComponentsTests/ViewComponentBuilderTests.cs]: Parameters are fixed.

**Pull request merged**